datajoint/autopopulate.py

```diff
@@ -96,7 +96,7 @@ def _jobs_to_do(self, restrictions):
 
     def populate(self, *restrictions, suppress_errors=False, return_exception_objects=False,
                  reserve_jobs=False, order="original", limit=None, max_calls=None,
-                 display_progress=False):
+                 display_progress=False, **kwargs):
```

Comment:
rather than using `**kwargs`, use an explicit named argument, something like `make_kwargs` containing the dict of all the custom parameters. This is because `populate` already has a lot of keyword arguments. If someone misspells one of the regular keyword arguments, then rather than raising an error, populate will simply pass the misspelled argument into `**kwargs`.

Reply:
I implemented your suggestion in the next commit #2d62835. Feel free to close the pull request if this feature does not align with your DataJoint philosophy 👍